src/test/scala/org/camunda/feel/impl/interpreter/InterpreterNumberExpressionTest.scala

```diff
@@ -180,11 +180,13 @@ class InterpreterNumberExpressionTest
     eval("null = 2") should be(ValBoolean(false))
     eval("null != 2") should be(ValBoolean(true))
 
-    eval("2 > null") should be(ValBoolean(false))
-    eval("null < 2") should be(ValBoolean(false))
+    eval("2 > null") should be(ValNull)
+    eval("null < 2") should be(ValNull)
+  }
 
-    eval("null in < 2") should be(ValBoolean(false))
-    eval("null in (2..4)") should be(ValBoolean(false))
+  ignore should "compare with 'null in'" in {
+    eval("null in < 2") should be(ValNull)
+    eval("null in (2..4)") should be(ValNull)
```

Comment:
:fast_forward: Let's create a follow-up issue to fix these cases as well as the unary-tests. :sweat_smile:

Reply:
https://github.com/camunda/feel-scala/issues/679